### ISSUE TITLE
[Bug] Plugin Unavailable in Grafana 9.1.7

### ISSUE THREAD

**Issue by hackdefendr** [grafana/grafana-infinity-datasource#424]
## **Describe the bug**

![image](https://user-images.githubusercontent.com/2224217/194350161-536a691a-d665-48e9-8a4e-25482c633217.png)

Per the above screen, simply installing the plugin without configuring *does not* work.  
Per the below screen, I have the plugin installed:

![image](https://user-images.githubusercontent.com/2224217/194350608-79edf172-439f-4959-9a0f-4ce5eb7a7d68.png)

## **Steps to reproduce**

1. Install Grafana 9.1.7 (latest)
2. Via the command line, run: `grafana-cli plugins install yesoreyeram-infinity-datasource`
3. Once successful, return to Grafana GUI
4. Go to datasources, add new, and find Infinity listed.
5. Per the documentation, no configuration is needed and the plugin should just work.
6. See first screenshot at the top of this bug report.

## **Version Details:**

- Grafana version : v9.1.7 (0cbb79298d)
- Plugin version : 1.0.0

## **Additional context**

If you fix this, I will sponsor you because Infinity is the only real way to accomplish what I am trying to accomplish.

Regards,
Jeff

**Comment by hackdefendr** [comment]
Sure I can do that.

I installed Grafana on a Redhat 8.6 VM using the posted documentation from their [website](https://grafana.com/grafana/download?pg=get&plcmt=selfmanaged-box1-cta1).  I have tested in the latest Firefox and Chrome both on Windows 10 Enterprise. The plugin was installed on the command line per the instructions on your [Wiki](https://sriramajeyam.com/grafana-infinity-datasource/wiki/installation). 

With no configuration, I see **Plugin Unavailable**, even though it is clearly available from the settings page.

**Comment by hackdefendr** [comment]
Is there anything else I can provide to help? We really need this plugin to work.

**Comment by hackdefendr** [comment]
You can easily spin up a CentOS 8 vm to test with. It is the same code
base as RH.

**Comment by hackdefendr** [comment]
How can I make this more of a priority?  I promised to sponsor you and I
hoped that would be enough.

We really need this plugin. None of the alternative plugins have the same
feature set as yours.

**Comment by hackdefendr** [comment]
I have a couple of servers where I could build a CentOS VM and host it for you. Or if it would help, I could try compiling a binary on my RH vm then send you the logs. I'm here to help however I can.

**Comment by hackdefendr** [comment]
Update: I have since upgraded to Grafana 9.2.3 and to plugin v1.1.0 and I am still getting Plugin Unavailable on Redhat 8. I'm not sure what proper executable permissions you are talking about. From what I can tell, everything is owned by root and has the executable bit set. Every other plugin I have installed is set the same way.

**Comment by hackdefendr** [comment]
A promise is a promise. I think you are doing good work here. Your plugin isn't like any of the others that claim to do the same things. I have requested some funds through my firm for software support and planned on sponsoring you with a little of that.

Besides, I highly doubt I would have resolved this without your suggestion. When I saw my /etc/fstab contained noexec for /var, my mouth fell open and I definitely said some words 😄
